### DISCUSSION TITLE
GIT branching strategy

### DISCUSSION THREAD

**Discussion by dapolach** [zabiny/klabis#237] [General]
Ahoj vyvojari :) - @pavelrotek @otahirs @saytaM12 @jakub-h @lufik 

Meli bysme se domluvit na tom jakou zvolime strategii pro GIT branching. 

Pokud si nejste jisti o co jde popr. jake jsou moznosti, tak docela pekne udelany prehled najdete [tady](https://phoenixnap.com/kb/git-branching-strategy). 

Za mne bych sel jednoduchym flow - idealne (prekvapive) Github flow. To vypada takto: 
![image](https://github.com/zabiny/klabis/assets/123363733/83f8ead6-5569-46c0-bc35-91d645d87a55)

Pokud by se casem ukazalo ze potrebujeme neco robustnejsiho, tak bych sel cestou Gitlab flow. Ale verim ze to nebude potreba :)

**Comment by otahirs** [comment]
nejsme kriticky system, taky bych to drzel jednoduche. main a feature branche (bez oddeleni devel/release) mi prijde ok

**Comment by pavelrotek** [comment]
Ahoj souhlas s otikem. Bude to uplne stacit.

-pr

čt 30. 11. 2023 v 11:32 odesílatel Otakar Hirš ***@***.***>
napsal:
